com.varnamproject.Varnam.json

```diff
@@ -0,0 +1,171 @@
+{
+    "app-id": "com.varnamproject.Varnam",
```

Comment:
Shouldn't it be `com.varnamproject.varnam-editor` if that's the component being shipped?

Reply:
It's both. The main utility is the CLI `varnam` invoked with `flatpak run com.varnnamproject.Varnam varnamc -s ml -t 'word'`. The editor builds on the CLI `varnamc`